models/claims_preprocessing/normalized_input/staging/normalized_input__stg_eligibility.sql

```diff
@@ -7,7 +7,11 @@
 
 select
       patient_id
-    , patient_id||coalesce(data_source,'')||coalesce(payer,'')||coalesce(plan,'')||coalesce(cast(enrollment_start_date as {{ dbt.type_string() }}),'')||coalesce(cast(enrollment_end_date as {{ dbt.type_string() }}),'') as patient_id_key
+    {% if target.type == 'fabric' %}
+        , patient_id+coalesce(data_source,'')+coalesce(payer,'')+coalesce("plan",'')+coalesce(cast(enrollment_start_date as {{ dbt.type_string() }}),'')+coalesce(cast(enrollment_end_date as {{ dbt.type_string() }}),'') as patient_id_key
```

Comment:
Same note here about using the concat macro.

Reply:
@sarah-tuva 
It -can- but still needs conditional logic anyway for the "plan" so I just left it.